drivers/wasapi/audio_driver_wasapi.cpp

```diff
@@ -67,8 +67,8 @@ Error AudioDriverWASAPI::init_device() {
 
 	switch (channels) {
 		case 2: // Stereo
-//		case 6: // Surround 5.1
-//		case 8: // Surround 7.1
+			//		case 6: // Surround 5.1
```

Comment:
clang-format did something ugly here :)

Reply:
So it seems. And it insists in formatting those lines like that. I've had to correct them manually.